tests/utils/dockercli/volumelifecycle.go

```diff
@@ -53,8 +66,70 @@ func KillDocker(ip string) ([]byte, error) {
 	return out, err
 }
 
+// GetVDVSPlugin - get vDVS plugin id
+func GetVDVSPlugin(ip string) (string, error) {
+	out, err := ssh.InvokeCommand(ip, dockercli.GetVDVSPlugin)
+	if misc.FormatOutput(out) == "" {
+		return "", fmt.Errorf("vDVS plugin unavailable")
+	}
+	return strings.Fields(misc.FormatOutput(out))[0], err
+}
+
+// StartVDVSPlugin - manually start vDVS plugin
+func StartVDVSPlugin(ip string) ([]byte, error) {
+	log.Printf("Manually starting vDVS plugin on VM [%s]\n", ip)
+	out, err := ssh.InvokeCommand(ip, dockercli.StartVDVSPlugin)
+	return out, err
+}
+
+// KillVDVSPlugin - kill vDVS plugin. It is restarted automatically
+func KillVDVSPlugin(ip, pluginID string) ([]byte, error) {
+	out, err := ssh.InvokeCommand(ip, dockercli.KillVDVSPlugin+pluginID)
+	time.Sleep(2 * time.Second)
+	return out, err
+}
+
+// RestartVDVSPlugin - kill vDVS plugin. It is restarted automatically
+// if autorestart failed, start it manually
+func RestartVDVSPlugin(ip string) ([]byte, error) {
```

Comment:
`RestartVDVSPlugin kill vDVS plugin. It is restarted automatically` .. leads to confusion .. 

something better name certainly not `RestartVDVSPlugin`

either come up with some intuitive name or let testcase to deal separately that make testcase more readable.